processing/src/main/java/org/apache/druid/query/QueryContext.java

```diff
@@ -179,6 +192,11 @@ public boolean getAsBoolean(
     return QueryContexts.getAsBoolean(parameter, get(parameter), defaultValue);
   }
 
+  public Integer getAsInt(final String parameter)
```

Comment:
I wonder, since we have methods for all of these on `QueryContexts`, and we use `QueryContexts` to get each specific value type, should we just omit these methods and standard on always using `QueryContexts` to get context values? Thoughts?